content/docs/standard-library/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Standard Library
```

Comment:
I suggest shorten path to just `std` so it's easier to remember and write. Title should remain the same tho